### ISSUE TITLE
[Rendering] Voiceover not announcing error message for checkbox on submit

### ISSUE THREAD

**Issue by JoshuaChingMSFT** [microsoft/AdaptiveCards#8008]
### Target Platforms

iOS

### SDK Version

1.4

### Application Name

Microsoft Teams

### Problem Description

Accessibility voice over does not announce checkbox error message on submit.

### Screenshots

![Uploading image.png…]()

### Card JSON

```JSON
{
    "body": [
        {
            "isSubtle": false,
            "text": " ",
            "type": "TextBlock",
            "maxLines": 0
        },
        {
            "isSubtle": false,
            "id": "Name",
            "text": "**title text**",
            "wrap": true,
            "type": "TextBlock",
            "maxLines": 0
        },
        {
            "isSubtle": false,
            "text": " ",
            "type": "TextBlock",
            "maxLines": 0
        },
        {
            "placeholder": "Enter one or more recipients",
            "isRequired": true,
            "id": "participants",
            "errorMessage": "Select at least one recipient",
            "choices": [
                {
                    "title": "Megan Bowen",
                    "value": "testvalue"
                }
            ],
            "label": "Send to",
            "type": "Input.ChoiceSet",
            "isMultiSelect": true
        },
        {
            "placeholder": "Enter description",
            "isRequired": true,
            "maxLength": 1000,
            "errorMessage": "This field can't be blank",
            "id": "service",
            "label": "Description",
            "type": "Input.Text",
            "regex": "^([^\"$]|\\$\\d+|\\$\\s)+$"
        },
        {
            "actions": [
                {
                    "title": "Send request",
                    "data": {
                        "providerId": "id",
                        "action": "submitAction"
                    },
                    "type": "Action.Submit"
                }
            ],
            "type": "ActionSet",
            "spacing": "ExtraLarge"
        }
    ],
    "speak": "speak test",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.4"
}
```

**Comment by VikrantSingh01** [comment]
@jwoo-msft : Can you take a look at this bug which is impacting many of the iOS accessibility issue.
